serialization.go

```diff
@@ -88,6 +122,28 @@ func SendJSON(w http.ResponseWriter, ans any) {
 	}
 }
 
+// SendError sends an error.
+// Declared as a variable to be able to override it for clients that need to customize serialization.
+var SendError = func(w http.ResponseWriter, r *http.Request, err error) {
+	accept := parseAcceptHeader(r.Header.Get("Accept"), nil)
+	if accept == "" {
+		accept = "application/json"
+	}
+
+	switch accept {
+	case "application/xml":
+		SendXMLError(w, r, err)
+	case "text/html":
+		_ = SendHTMLError(r.Context(), w, err)
+	case "text/plain":
+		_ = SendText(w, err)
+	case "application/json":
+		SendJSONError(w, err)
+	default:
```

Comment:
So in the case of yaml we'll just want to respond with json?

Reply:
Adding yaml here too